### PR TITLE
orm.js updated with model for UserAnswers

### LINKED ISSUE #72: Add UserAnswers model and DB schema to orm.js
A a new model in orm.js based on this definition

#### `user_answers` table / UserAnswers model

All of these columns are `NOT NULL` unless otherwise specified.

- `id`: primary key / integer sequence
- `user_id`: string (foreign key to `users.id`)
- `question_number`: integer
- `is_correct`: [boolean](https://sequelize.org/master/manual/model-basics.html#boolean)
-  `elapsed_time_ms`: integer
- `created_at` : timestamp (we don't need updated_at since this is immutable. this name can be [configured in the sequelize model](https://sequelize.org/master/manual/model-basics.html#timestamps) as we don't want the default camel case name)

Please use the [sequelize docs](https://sequelize.org/master/)

### PR DESCRIPTION
**Pull request by ambikasadhu1101**
Fixes #72

### CODE REVIEW

**Review by vegetabill — CHANGES_REQUESTED** [review]
This looks great! 

In addition to the two small revision comments below, can you export UserAnswers at the bottom of the file as well? If you need any help revising the PR, ping me or any of the mentors.

Thanks!

**Comment by ambikasadhu1101** [comment]
Sure, I'll just go through the above

File: JS/orm.js [hunk 1]
```diff
@@ -22,7 +22,38 @@ User.init(
     updatedAt: "updated_at",
   }
 );
-
+class UserAnswers extends Model{}
```
[the remark this answers is not in the document]
Comment (ambikasadhu1101) [reply JS/orm.js:25]: I am a bit confused here. In my opinion both are class based?

[the remark this answers is not in the document]
Comment (vegetabill) [reply JS/orm.js:25]: Sorry I misread the line! Just ignore this.

**Review by vegetabill — APPROVED** [review]
Revised version looks great! Thanks for being the first PR of OSD!!

**Pull request merged**